controllers/githubactionrunner_controller.go

```diff
@@ -35,6 +36,8 @@ import (
 	garov1alpha1 "github.com/evryfs/github-actions-runner-operator/api/v1alpha1"
 )
 
+const POOL_LABEL = "garo.tietoevry.com/pool"
```

Comment:
Codacy found an issue: [exported const POOL_LABEL should have comment or be unexported](https://app.codacy.com/gh/evryfs/github-actions-runner-operator/pullRequest?prid=5969435)